n2y/blocks.py

```diff
@@ -388,6 +392,44 @@ def to_pandoc(self):
         return Row(('', [], []), cells)
 
 
+class ColumnListBlock(Block):
+    def __init__(self, client, notion_data, page, get_children=True):
+        super().__init__(client, notion_data, page, get_children)
+
+    def to_pandoc(self):
+        cells = self.children_to_pandoc()
+        colspec = [(AlignDefault(), ColWidthDefault()) for _ in range(len(cells))]
+        table = Table(
+            ('', [], []),
+            Caption(None, []),
+            colspec,
+            TableHead(('', [], []), []),
+            [TableBody(
+                ('', [], []),
+                RowHeadColumns(0), [],
+                [Row(('', [], []), cells)])],
+            TableFoot(('', [], []), [])
+        )
+        return table
+
+
+class ColumnBlock(Block):
+    def __init__(self, client, notion_data, page, get_children=True):
+        super().__init__(client, notion_data, page, get_children)
+
+    def to_pandoc(self):
+        string_of_children = repr(self.children_to_pandoc())
```

Comment:
Are these lines trying to replace each `Para` call with a `Plain`?

If so, why are we trying to do that? Is it because we can't support paragraphs within a table in pandoc's data model?

If we can't support paragraphs in a table, then maybe we should reconsider upgrading the column block like this, since I think Paragraph blocks in a ColumnBlock is going to be pretty common.

I am surprised that we would need to use an `eval` call here? (In general, `eval` is very dangerous and should be avoided.) I bet there would be a way to swap the Para with Plain without using eval.

Reply:
I ran into an error earlier but I can't replicate it now so the `Para` wasn't the issue. I had no clue about `eval`, I'll look into that.

Reply:
ast.literal_eval is safer, right?